CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 ## Unreleased
+
+## 1.20.9
+* Fixed an issue where playbook_unreachable_condition validation failed falsely.
```

Comment:
```suggestion
* Fixed an issue where **validate** falsely failed with error `PB101` and `PB123` <because...>
```
what is the reason it falsely failed?